src/types/general.ts

```diff
@@ -49,3 +49,29 @@ export function getDateTimeFromString(dateTime: string): DateTime {
   const dt = dateTime.trim();
   return dt.trim().length > 0 ? DateTime.fromISO(dt) : DateTime.now();
 }
+
+// Type-safe transformation function with runtime validation that ensures
+// that raw ISO date time stamps are transformed into valid ts-luxon DateTime
+// instances.
+export const transformEntityDates = (data: any): any => {
```

Comment:
This transformation function is new and is used by some of the entity *List hooks/API functions. Since this is new, definitely worth checking over to see if it's easy to use and implemented well.